src/molecules/Carousel.tsx

```diff
@@ -19,13 +19,9 @@ export const Carousel: React.FC<CarouselProps> = ({ images }) => {
   }
 
   return (
-    <Box marginHorizontal={1.4}>
+    <Box marginHorizontalMd="1.4">
       <Image src={primaryImage.fullImage} />
-      <Composition
-        gap={0.2}
-        templateCols="1fr 1fr 1fr 1fr 1fr 1fr"
-        marginVertical={0.2}
-      >
+      <Composition gap="3px" marginVertical="3px" templateCols="repeat(6, 1fr)">
```

Comment:
Also pay attention that at the moment it will always give you 6 columns, stretching the column width (`1fr`). 

Maybe consider `repeat(auto-fill, 100px)`, where `100px` is a fixed thumbnail (small image) size. This _may_ be what you want. Just experiment with responsive and how those columns behave.